config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -300,46 +252,40 @@ presubmits:
       testgrid-num-failures-to-alert: "10"
       testgrid-dashboards: google-gce
     labels:
-      preset-service-account: "true"
       preset-k8s-ssh: "true"
-      preset-pull-kubernetes-e2e: "true"
-      preset-pull-kubernetes-e2e-gce: "true"
+      preset-dind-enabled: "true"
     decorate: true
     decoration_config:
-      timeout: 105m
+      timeout: 110m
     path_alias: k8s.io/kubernetes
     extra_refs:
     - org: kubernetes
-      repo: release
+      repo: kops
       base_ref: master
-      path_alias: k8s.io/release
+      path_alias: k8s.io/kops
     spec:
+      serviceAccountName: k8s-kops-test
       containers:
         - command:
             - runner.sh
-            - /workspace/scenarios/kubernetes_e2e.py
           args:
-            - --build=quick
-            - --cluster=
-            - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.7.5
-            - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.1.9
-            - --env=LOG_DUMP_SYSTEMD_SERVICES=containerd
-            - --env=CONTAINER_RUNTIME_TEST_HANDLER=true
-            - --env=KUBE_MASTER_OS_DISTRIBUTION=ubuntu
-            - --env=KUBE_GCE_MASTER_IMAGE=ubuntu-2204-jammy-v20220712a
-            - --env=KUBE_GCE_MASTER_PROJECT=ubuntu-os-cloud
-            - --env=KUBE_NODE_OS_DISTRIBUTION=ubuntu
-            - --env=KUBE_GCE_NODE_IMAGE=ubuntu-2204-jammy-v20220712a
-            - --env=KUBE_GCE_NODE_PROJECT=ubuntu-os-cloud
-            - --extract=local
-            - --gcp-master-image=ubuntu
-            - --gcp-node-image=ubuntu
-            - --gcp-zone=us-west1-b
-            - --ginkgo-parallel=30
-            - --provider=gce
-            - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-canary
-            - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
-            - --timeout=80m # thinking about making this longer? don't! 80m is a hard cap, and should get down to no more than 60m.
+            - bash
+            - -c
+            - |
+              ARGS="--set=spec.containerd.runc.version=1.1.9 --set=spec.containerd.version=1.7.5"
```

Comment:
runc v1.1.9 support needs to be added. See https://github.com/kubernetes/kops/pull/16043.
Maybe also switch to containerd v1.7.7.